src/Entities/Comment.php

```diff
@@ -8,4 +8,5 @@ class Comment
     public string $comment;
     public string $date_created;
     public int $issue_id;
+
```

Comment:
Mysteriously appearing newline in entity

Reply:
done